src/plugins/chart_expressions/expression_xy/common/expression_functions/xy_vis.ts

```diff
@@ -190,6 +190,12 @@ export const xyVisFunction: ExpressionFunctionDefinition<
         defaultMessage: 'Show values in legend',
       }),
     },
+    xAxisInterval: {
```

Comment:
@VladLasitsa, will we have the same option at the `layeredXyVis`? If so, should it be inside the `dataLayer` expression function? Just curious about it.

And also, we should definitely add the validation of the `xAxisInterval`. For now, it is not very usable for the end-users, I think. 

WDYT?

Reply:
As we have only one x-axis we should add this arg directly in `xyVis` and `layeredXyVis`. 

Added validation.